TechnologyAssessmentApp/TechnologyAssessmentApp/Models/ArticleViewModel.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ArticleViewModel.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 24/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import Foundation
+
+struct ArticleViewModel: Codable {
```

Comment:
This look like a `Model` and not a `ViewModel`.
Ask yourself what does this `struct` does ? From my understand, it represent a struct which is the root result of JSON response. It's nothing related to `Article`. maybe `Result` is a better naming here ?